state-chain/chains/src/sol/instruction_builder.rs

```diff
@@ -139,6 +138,55 @@ impl SolanaInstructionBuilder {
 
 		Self::finalize(instructions, nonce_account.into(), agg_key.into(), compute_price)
 	}
+
+	pub fn ccm_transfer(
```

Comment:
Rename this to `ccm_transfer_native`

Reply:
This will support other assets within itself - so we can just call this function for both native and token asset.

Reply:
right, makes sense :+1: